pkg/nfd-worker/nfd-worker.go

```diff
@@ -281,6 +281,10 @@ func (w *nfdWorker) Run() error {
 
 		case <-configTrigger:
 			w.configure(configFilePath, w.args.Options)
+
+			// Always re-label after a re-config event. This way the new config
+			// comes into effect even if the sleep interval is long (or infinite)
+			labelTrigger = time.After(1 * time.Second)
```

Comment:
Should we use `time.After(0)` here?

Reply:
I'm not sure. The idea behind this was a 1 second grace period/rate limiter, e.g. if there were multiple filesystem operations in a quick succession, each of which would cause an (unnecessary) config reload. So I'd probably keep it this way. But, a comment describing this would be in order. WDYT?

Reply:
With a fresh look, you're correctl. `0` is better here. I'll change it. We do the rate limiting elsewhere.

Reply:
Changed to `0`